### ISSUE TITLE
qhull_VERSION incorrection.

### ISSUE THREAD

**Issue by zheddie** [qhull/qhull#115]
In latest code of CMakeLists.txt, the qhull_VERSION have something like "8.1-alpha1", which would break the Makefile, when it try to grep it from CMakeLists.txt.

**Comment by zheddie** [comment]
The detail error message is as below:
```
gcc -shared -lm -o 1  src/libqhullstatic_r/global_r.o src/libqhullstatic_r/stat_r.o src/libqhullstatic_r/geom2_r.o src/libqhullstatic_r/poly2_r.o src/libqhullstatic_r/merge_r.o src/libqhullstatic_r/libqhull_r.o src/libqhullstatic_r/geom_r.o src/libqhullstatic_r/poly_r.o src/libqhullstatic_r/qset_r.o src/libqhullstatic_r/mem_r.o src/libqhullstatic_r/random_r.o src/libqhullstatic_r/accessors_r.o src/libqhullstatic_r/usermem_r.o src/libqhullstatic_r/userprintf_r.o src/libqhullstatic_r/io_r.o src/libqhullstatic_r/user_r.o src/libqhullstatic_r/rboxlib_r.o src/libqhullstatic_r/userprintf_rbox_r.o
# the following line fails under MSYS, not needed for SO=dll
(cd lib/ && ln -f -s libqhull_r.so.8.1 1 libqhull_r.so)
ln: target 'libqhull_r.so': No such file or directory
make: [Makefile:749: 1] Error 1 (ignored)
(cd lib/ && ln -f -s libqhull_r.so.8.1 1 libqhull_r.so.8.1)
ln: target 'libqhull_r.so.8.1': Not a directory
make: [Makefile:750: 1] Error 1 (ignored)
```

**Comment by cbbarber** [comment]
Thanks for your pull request.  I've merged #115
